src/Pagination/PaginateDirective.php

```diff
@@ -43,6 +44,14 @@ public static function definition(): string
   """
   builder: String
```

Comment:
Let's also amend this and `model` with a hint that they are mutually exclusive with each other.